tests/integration/web/info/info_test.py

```diff
@@ -119,15 +119,11 @@ def fetch_results(self):
 
 @pytest.fixture
 def new_location(db):
-    location = Location(id="testlocation")
-    location.save()
+    location = Location(id="mylocation")
```

Comment:
This was not how it was meant, you can simply use `mylocation`/`myroom` in the places where a location/room id is needed, you don't need to create fixtures for them

Reply:
I have changed the test code to use "myroom" and "mylocation" id's in the url setups.